frontend/src/components/studySamples/CompletedSamplesTable.tsx

```diff
@@ -67,10 +71,26 @@ const COMMENT: CompletedSampleColumn = {
 }
  
 interface CompletedSamplesTableProps {
-	completedSamples: CompletedStudySample[]
+	studyID: FMSId
+	step: StudySampleStep
+	tableState?: CompletedSamplesTableState
+	settings?: StudyUXStepSettings
+	workflowAction: WorkflowActionType
 }
 
-function CompletedSamplesTable({completedSamples} : CompletedSamplesTableProps) {
+function CompletedSamplesTable({studyID, step, workflowAction, tableState, settings} : CompletedSamplesTableProps) {
+	const dispatch = useAppDispatch()
+
+	const pageSize = settings?.pageSize ?? 10
```

Comment:
This should use a front-end wide default constant instead of a page specific value (10) i think.